modules/renter/persist.go

```diff
@@ -446,3 +458,17 @@ func (r *Renter) LoadSharedFilesASCII(asciiSia string) ([]string, error) {
 	dec := base64.NewDecoder(base64.URLEncoding, bytes.NewBufferString(asciiSia))
 	return r.loadSharedFiles(dec)
 }
+
+func (r *Renter) updatePersistVersionFrom040To133() error {
+	metadata := persist.Metadata{
+		Header:  settingsMetadata.Header,
+		Version: persistVersion040,
+	}
+
+	err := persist.LoadJSON(metadata, &r.persist, filepath.Join(r.persistDir, PersistFilename))
+	if err != nil {
+		return err
+	}
+	metadata.Version = persistVersion133
+	return persist.SaveJSON(metadata, r.persist, filepath.Join(r.persistDir, PersistFilename))
```

Comment:
Part of the upgrade is setting the new values that were added to their default settings.